pkg/service/model_build_listener.go

```diff
@@ -20,17 +20,17 @@ func (t *defaultModelBuildTask) buildListeners(ctx context.Context, scheme elbv2
 	}
 
 	for _, port := range t.service.Spec.Ports {
-		_, err := t.buildListener(ctx, port, *cfg, scheme)
+		_, err := t.buildListener(ctx, t.service, port, *cfg, scheme)
 		if err != nil {
```

Comment:
we don't need to pass t.service down the call stack, we can just access t.service within `buildTargetGroupHealthCheckPort` :D 
(the reason we have this defaultModelBuildTask is to save states(e.g. service object been handled) as variables into defaultModelBuildTask instead of pass them around).